doc/source/templates/04_finetuning_llms_with_deepspeed/compute_configs/aws_70b_v1.yaml

```diff
@@ -0,0 +1,27 @@
+
+
+
+
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
```

Comment:
These cannot have cloud id and region (this is not a valid yaml file and will error when product tries to parse it).